fudge-ratio/src/main/webapp/code/time.js

```diff
@@ -1,52 +1,46 @@
 class Timer {
   seconds = 0;
```

Comment:
This is a class I created to capture Jerry's logic, but in this PR I proposed what I think is a better approach:
 - Instead of storing the seconds, minutes and hours, and each time we change them check they are still consistent (0 <= seconds < 60...) we not just store the number of seconds and JUST when we need to display the information do the calculations?

Reply:
Yeah, I agree. It is much simpler now. I think we should use your changes but ai want to hear from Jerry since he wrote the original

Reply:
I agree with the modifications in the PR.